buildSrc/src/main/java/dependencies/Dep.kt

```diff
@@ -85,6 +86,7 @@ object Dep {
         val core = "com.google.firebase:firebase-core:16.0.4"
         val fireStore = "com.google.firebase:firebase-firestore:17.1.3"
         val auth = "com.google.firebase:firebase-auth:16.0.5"
+        val crashlytics = "com.crashlytics.sdk.android:crashlytics:2.9.8"
```

Comment:
I can not receive 🤔 
![image](https://user-images.githubusercontent.com/1386930/51076193-94ff0a00-16d8-11e9-9d35-39994c9469b6.png)

https://stackoverflow.com/a/53750745/4339442
I don't know why but, can you downgrade?

Reply:
I received crash! Thanks!👍
I can not select that... Anyway it works!
![screenshot_20190113-023409 2](https://user-images.githubusercontent.com/1386930/51076450-eb217c80-16db-11e9-9498-ada69fedcfe8.png)